utils/fsHandler.js

```diff
@@ -0,0 +1,19 @@
+import fs from 'fs';
+
+const createFolderIfNotExists = path => {
+  return new Promise((resolve, reject) => {
+    fs.mkdir(path, err => {
```

Comment:
This code won't work if there you are creating folder inside folder. You need to add recursive flag.

Reply:
added recursive flag and set it to true.